src/protocols/fp-meta.ts

```diff
@@ -0,0 +1,137 @@
+import { FPEnvelopeTurnaround, FPEnvelopeType, FPStoreType } from "./fp-envelope";
+import { CID } from "multiformats";
+
+export interface FPGetMetaReq {
+    readonly cid: CID;
+}
+/*
+ * A request to get a META file.
```

Comment:
meta isn't quite a file, its an API response, as it must be composed by the server dynamically depending on current state